### DISCUSSION TITLE
Disable hovering over individual cells.

### DISCUSSION THREAD

**Discussion by monolith04** [Akascape/CTkTable#54] [Q&A] [answered]
Hi, I have a table with 2 columns and 3 rows, when the cursor is placed on any cell of the table, the color of the cell changes. 
**Question:** how to make the cells in the first column not change color when hovering, but only in the second?

**Comment by monolith04** [comment, marked as answer]
I'm solved this with: 
`edit_column(column=0, hover=False)`

**Comment by Akascape** [comment]
@monolith04 
>how to make the cells in the first column not change color when hovering, but only in the second?

Use `table.edit_column(0, hover=False)` to disable the hover in first column.

Or use  `table.edit_column(1, hover=True)` to enable hovering in second column only.
